ibis/expr/types/relations.py

```diff
@@ -305,26 +305,29 @@ def view(self) -> Table:
 
         return ops.SelfReference(self).to_expr()
 
-    def difference(self, right: Table, distinct: bool = True) -> Table:
-        """Compute the set difference of two table expressions.
+    def difference(self, *tables: Table, distinct: bool = True) -> Table:
+        """Compute the set difference of multiple table expressions.
 
         The input tables must have identical schemas.
 
         Parameters
         ----------
-        right
-            Table expression
+        *tables
```

Comment:
this is a breaking change?

Reply:
See my comment below, I think we can address this pretty easily.

Reply:
Ok, we now accept the `right` kwarg still but raise a `FutureWarning`. This should be resolved.